### DISCUSSION TITLE
Clarett+ 8Pre Question

### DISCUSSION THREAD

**Discussion by DonBrown2** [geoffreybennett/alsa-scarlett-gui#104] [General]
I followed the instructions to install alsa-scarlett-gui, and everything went as expected.  Great job by the way.  I did notice that the LINE/INST buttons show up for the first two analogue inputs but are blank for the 3-8.  I don't see anything in the documentation that might explain why these are blank.  Everything is functional and the AIR function is working so its not a big issue.  Just curious if you could point me toward a fix if one exists.   Thanks again for your work on this.  It is very much appreciated.

**Comment by geoffreybennett** [comment]
Hi Don,

Thank you :).

Regarding the Line/Inst buttons missing for Inputs 3-8, it's because those inputs have no such functionality, so there's nothing to fix.

The front two inputs are Combo XLR jacks supporting Mic/Line/Inst inputs, and the rear six inputs are Combo XLR jacks supporting Mic/Line inputs. It's in the Focusrite Clarett+ 8Pre User Guide https://fael-downloads-prod.focusrite.com/customer/prod/s3fs-public/downloads/Clarett%2B%208Pre%20User%20Guide%20V2.pdf pages 8, 10, and 23.

If you get a chance, please test the dev branch of alsa-scarlett-gui:

![image](https://github.com/geoffreybennett/alsa-scarlett-gui/assets/1866657/0e6ad2bf-25aa-4744-b0a5-11601ecd3a82)

Let me know if you have any other questions; I'm happy to help!

Thanks,
Geoffrey.

**Reply by DonBrown2** [reply]
Wasn't expecting a response tonight but it was nice to hear the feedback.  I just picked up the 8Pre and perused the manual just to get the firmware update and basic config done.  Thanks for pointing out the user manual areas to review.   I would be happy to test the dev branch of your software as well.  Your work is impressive.  Do you have a donation portal specifically for your efforts?  I'd like to contribute. My apologies if that is documented elsewhere.  I play whackamole a lot and sometimes its just quicker to ask.

**Reply by geoffreybennett** [reply]
Hi Don,

Thank you! To test the dev branch, please follow the instructions at https://github.com/geoffreybennett/alsa-scarlett-gui/blob/master/INSTALL.md but before running the `make -j4` command, switch to the dev branch with `git checkout dev`.

Donations are greatly appreciated; please see https://github.com/geoffreybennett/alsa-scarlett-gui#donations for my PayPal and Liberapay links.

Regards,
Geoffrey.

**Comment by geoffreybennett** [comment]
To clarify, if you previously built it from a github clone, then you can just do:
```
cd alsa-scarlett-gui/src
git checkout dev
make -j4
```
To switch back to the old version in case of any issues you can do `git checkout master` and `make` again.

**Comment by DonBrown2** [comment]
I am getting an error on the #include openssl/sha.h: No such file or
directory.  Do you require a specific openssl package?

**Comment by geoffreybennett** [comment]
For Fedora you need openssl-devel. In Ubuntu it's called libssl-dev I believe.

**Comment by DonBrown2** [comment]
Thanks.  Dev is up and running now.
